src/components/interfaces/HMI_API.xml

```diff
@@ -4179,6 +4179,40 @@
     </param>
   </function>
 
+  <function name="GetFileFromHMI" messagetype="request">
+    <description>This request is sent to retrieve a file from the HMI.</description>
+    <param name="fileName" type="String" maxlength="255" mandatory="true">
+		  <description>File name that should be retrieved.</description>
+	  </param>
+    <param name="fileType" type="Common.FileType" mandatory="false">
+		  <description>Selected file type.</description>
+	  </param>
+    <param name="appServiceId" type="String" mandatory="false">
+		  <description> ID of the service that should have uploaded the requested file.</description>
+	  </param>
+    <param name="offset" type="Integer" minvalue="0" maxvalue="2000000000" mandatory="false">
```

Comment:
Not certain whether `offset`, `length`, and `fileType` are necessary to pass through. From my understanding, isn't this RPC solely for getting the full file path?

Reply:
removing offset and length but keeping Filetype. The HMI could probably decide to limit which filetype can be processed.

Reply:
Yeah, keeping `fileType` probably won't hurt, just not certain if it would be useful. Your call on this matter